x-pack/plugins/observability_solution/inventory/public/components/entities_grid/index.tsx

```diff
@@ -227,19 +273,20 @@ export function EntitiesGrid({
           );
         case ENTITY_DISPLAY_NAME:
           return (
-            <EuiLink
-              data-test-subj="inventoryCellValueLink"
-              className="eui-textTruncate"
-              href={getEntityRedirectUrl(entity)}
-            >
-              {entity[columnEntityTableId]}
+            <EuiLink data-test-subj="inventoryCellValueLink" href={getEntityRedirectUrl(entity)}>
+              <EuiFlexGroup gutterSize="s" alignItems="center">
+                <EuiFlexItem grow={0}>{getEntityIcon(entity)}</EuiFlexItem>
+                <EuiFlexItem className="eui-textTruncate">
+                  <span className="eui-textTruncate">{entity[columnEntityTableId]}</span>
+                </EuiFlexItem>
+              </EuiFlexGroup>
```

Comment:
Maybe we should have a component that returns the icon and the name? That's because if `getEntityIcon` returns null we'll still render an empty `EuiFlexItem` on the page.

Reply:
Hey @iblancof,

Quick Q - are we able to use the same logic that is used by the Hosts feature for consistency?

If we're not, it's OK - it's tech preview and a separate experience so it's not critical this is consistent right now.

cc @kpatticha @cauemarcondes

Reply:
> Quick Q - are we able to use the same logic that is used by the Hosts feature for consistency?
> 
> If we're not, it's OK - it's tech preview and a separate experience so it's not critical this is consistent right now.

Hi @roshan-elastic,

This is what I brought up with @cauemarcondes and @kpatticha this morning, and it’s outlined in the description. We can’t apply the same logic, so they suggested proceeding with the described behavior to unblock the feature.

Reply:
**Update on resolution:** This has been clarified and we've all agreed to accept potential inconsistencies for the sake of simplicity.